### PR TITLE
Additions for the origins taxonomy

### PR DESCRIPTION
**Pull request by teolemon**
Small additions for the origins taxonomy

### CODE REVIEW

File: taxonomies/origins.txt [hunk 1]
```diff
@@ -122,6 +123,24 @@ en:Sicily
 fr:Sicile
 it:Sicilia
 
+# Spanish regions
+
+<en:spain
+es:Andalucía
```
Comment (stephanegigandet) [inline taxonomies/origins.txt:129]: for origins, please always set an en: entry as the main entry (as it's done for French regions already).
If there isn't an English "translation", just use whatever the wikipedia page is called. e.g. https://en.wikipedia.org/wiki/Castilla%E2%80%93La_Mancha

Comment (teolemon) [inline taxonomies/origins.txt:129]:
```suggestion
en:Andalusia
es:Andalucía
```

File: taxonomies/origins.txt [hunk 2]
```diff
@@ -122,6 +123,24 @@ en:Sicily
...
+fr:Andalousie
+
+<en:spain
+es:Castilla-La Mancha
```
Comment (teolemon) [inline taxonomies/origins.txt:133]:
```suggestion
en:Castilla-La Mancha
es:Castilla-La Mancha
```

File: taxonomies/origins.txt [hunk 3]
```diff
@@ -122,6 +123,24 @@ en:Sicily
...
+
+<en:spain
+es:Galicia
+
+<en:spain
+es:Castilla y León
+
+<en:spain
+es:Navarra
+fr:Navarre
```
Comment (teolemon) [inline taxonomies/origins.txt:143]:
```suggestion
<en:spain
en:Galicia
es:Galicia

<en:spain
en:Castile and León
es:Castilla y León

<en:spain
en:Navarra
es:Navarra
fr:Navarre
```

File: taxonomies/origins.txt [hunk 4]
```diff
@@ -83,10 +88,26 @@ fr:Camargue
...
+<en:france
+en:Pays de la Loire
+fr:Pays de la Loire
+
+<en:france
+en:Auvergne
+fr:Auvergne
+
+<en:france
+en:Franche-Compte
```
Comment (stephanegigandet) [inline taxonomies/origins.txt:108]: Franche-Comté

File: taxonomies/origins.txt [hunk 5]
```diff
@@ -115,13 +136,125 @@ fr:Gers
...
+en:Périgord
+fr:Périgord
+
+<en:France
+en:Dordogne
+fr:Dordogne
+
+<en:France
+en:Guérande
+fr:Guérande, 44350 Guérande
```
Comment (stephanegigandet) [inline taxonomies/origins.txt:165]:
```suggestion
fr:Guérande
```

We'll need to handle that in another way, not through the taxonomy.

File: taxonomies/origins.txt [hunk 6]
```diff
@@ -83,10 +88,26 @@ fr:Camargue
...
+en:Pays de la Loire
+fr:Pays de la Loire
+
+<en:france
+en:Auvergne
+fr:Auvergne
+
+<en:france
+en:Franche-Compte
+fr:Franche-Compte
```
Comment (teolemon) [inline taxonomies/origins.txt:109]:
```suggestion
en:Franche-Comté
fr:Franche-Comté
```

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**